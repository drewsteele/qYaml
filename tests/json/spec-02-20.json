{"canonical": 1230.15, "exponential": 1230.15, "sexagesimal": 1230.15, "fixed": "1,230.15", "negative infinity": -Infinity, "not a number": NaN}