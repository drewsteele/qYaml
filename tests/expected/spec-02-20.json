{"canonical": 1230.15, "exponential": 1230.15, "fixed": 1230.15, "negative infinity": -Infinity, "not a number": NaN}
